pytest_html/plugin.py

```diff
@@ -344,14 +381,21 @@ def generate_summary_item(self):
             unicode_doc = unicode_doc.decode('utf-8')
         return unicode_doc
 
-    def _save_report(self, report_content, self_contained_html):
+    def _save_report(self, report_content):
         dir_name = os.path.dirname(self.logfile)
-        if not os.path.exists(dir_name):
-            os.makedirs(dir_name)
+        assets_dir = os.path.join(dir_name, 'assets')
```

Comment:
We hard-code this in a few places - do you think it would make sense to add it to the `HTMLReport` as an attribute? We could then just use `self.assets_path` throughout.